frontend/packages/citizen-frontend/src/Footer.tsx

```diff
@@ -0,0 +1,45 @@
+import React from 'react'
+import { useTranslation } from '~localization'
+import styled from 'styled-components'
+
+export default React.memo(function Footer() {
+  const t = useTranslation()
+  return (
+    <FooterContainer>
+      <MunicipalityContainer>{t.footer.espooLabel}</MunicipalityContainer>
+      <Nav>
+        <NavItem href={t.footer.privacyPolicyLink} data-qa={'nav-old-map'}>
+          {t.footer.privacyPolicy}
+        </NavItem>
+        <NavItem href={t.footer.sendFeedbackLink} data-qa={'nav-old-map'}>
+          {t.footer.sendFeedback}
+        </NavItem>
+      </Nav>
+    </FooterContainer>
+  )
+})
+
+const FooterContainer = styled.div`
+  display: flex;
+  flex-direction: row;
+  justify-content: center;
+  padding: 20px 0 20px 0;
+  font-size: 0.75rem;
+
+  & div {
+    display: inline-block;
+    margin: 0 40px 0 40px;
+  }
+`
+
+const MunicipalityContainer = styled.div``
+
+const Nav = styled.nav`
+  & a {
+    display: inline-block;
+    margin: 0 40px 0 40px;
+    text-decoration: none;
+  }
```

Comment:
These styles could just be moved under `NavItem`, which could be named something else since like I mentioned in my previous comment, they're not really a part of the application's navigation.